styleguide/_themes/derek/scss/patterns/grid/productFeatures.scss

```diff
@@ -30,11 +31,11 @@
 }
 
 @media only screen and (max-width: $screen-sm) {
-  .features {
+  .features-container {
     display: block;
     margin-top: 10px;
 
-    .desc {
+    .features-desc {
```

Comment:
On second thought since this is a bug fix and not a refactor task, maybe we shouldn't risk introducing new bugs into the system with a lot of refactoring.  @amberRrucker @bighappyface thoughts?